tests/requirements.txt

```diff
@@ -5,3 +5,4 @@ boto==2.9.0
 tox
 dopy
 PyCrypto
+openshift
```

Comment:
why openshift?

Reply:
It's needed for the k8s ansible tasks in the packet-ci playbook . Otherwise I'll have to rewrite the playbook. The requirement is only for test and not generally for Kubespray